features/step_definitions/helper_services.rb

```diff
@@ -503,7 +503,11 @@
   _project = project("iscsi-target", switch: false)
   if !_project.exists?(user:admin, quiet: true)
     @result = admin.cli_exec(:create_namespace, name: 'iscsi-target')
-    raise 'failed to "iscsi-target" project' unless @result[:success]
+    raise 'failed to create "iscsi-target" project' unless @result[:success]
+    @result = admin.cli_exec(:label, resource: 'ns/iscsi-target', key_val: 'security.openshift.io/scc.podSecurityLabelSync=false', overwrite: 'true')
```

Comment:
We only need this for 4.12+, perhaps add a check for before adding the labels ?
```
if env.version_ge("4.12", user: user)
```

Reply:
Good suggestion! Sorry for that, I'm not familar with ruby. Thanks for the details!